doc/development.rst

```diff
@@ -6,30 +6,31 @@ Nilearn development process
     :depth: 2
     :local:
 
+# Ideally this should go directly on the web pag
+
 How to help?
 =============
 
-* You are new to python and you don't know how to do xy
-
-   - Create a question on `neurostars <https://neurostars.org/>`_
-
-* If you discovered a bug, but don't know how to fix it
+* If you have a usage question : create a question on `neurostars <https://neurostars.org/>`_ with tag nilearn
 
-   - Create `an issue <https://github.com/nilearn/nilearn/issues>`_
+* You think you discovered a bug ? Create `an issue <https://github.com/nilearn/nilearn/issues>`_
+  including a minimal runnable example to showcase it (preferably using Nilearn data).
 
-* If you discovered a bug and know how to fix it, but don't know how to
-  get your code onto github (ie, you have some Python experience but have
-  never have used git/github, or have never written tests)
+* If you have ideas for new features, check it's in the scope of the project
+  and feel free to open an issue to discuss it.
 
-    - Learn git and github: http://try.github.io/
-    - Learn what tests are how to run them locally
-      (https://docs.pytest.org)
-    - Learn how to write doc/examples and build them locally
-      https://sphinx-gallery.github.io/
+* You think documentation can be improved, you can directly send proposed
+  improvements in `a pull request <https://github.com/nilearn/nilearn/pulls>`_.
 
-* You want to contribute code
-
-    - See below
+* Want to contribute code ? Thank you very much! For new features, please be
+  sure to create an issue first, to discuss whether it can be included and its
+  specifications. If you have time to help dealing with known issues check
+  `Good first issues <https://github.com/nilearn/nilearn/labels/Good%20first%20issue>`_
+  to get started, `known bugs <https://github.com/nilearn/nilearn/labels/Bug>`_
+  or `proposed enhancements <https://github.com/nilearn/nilearn/labels/Enhancement>`_.
+  In any case, before sending code, be sure to ** read and follow the
```

Comment:
Especially for new contributors, I think we should recommend that folks at least comment on the original issue to make sure that opening a PR to address it won't conflict with any existing PRs.